README.md

```diff
@@ -4,14 +4,158 @@
 
-This is a fork of [ensky/taiga-contrib-ldap-auth](https://github.com/ensky/taiga-contrib-ldap-auth) which also retrieves the various contributions and other forks into one.
+This is a fork of [ensky/taiga-contrib-ldap-auth](https://github.com/ensky/taiga-contrib-ldap-auth), which also retrieves the various contributions and other forks into one.
 
+## :whale: Installation & Configuration with Docker
 
-## Installation
+If you installed a dockerized Taiga using the 30 Minute Setup approach, you should be able to install this plugin using this guide.
+
+The following will assume that you have a clone of the [kaleidos-ventures/taiga-docker](https://github.com/kaleidos-ventures/taiga-docker) repository on the computer you want to host Taiga on.
+
+### taiga-back
+
+1. Edit the `taiga-back` section in the `docker-compose.yml`: Replace `image: taigaio/taiga-back:latest` with `build: ./custom-back`
+2. Create a folder `custom-back` next to the `docker-compose.yml` file
+3. In this folder, create a file `config.append.py`. The contents of the file are collapsed below.
+4.  In this folder, also create a `Dockerfile`. The contents of the file are collapsed below.
+
+If you were to start Taiga now, it would not pull the `taiga-back` directly from Docker Hub but instead build the image from the specified `Dockerfile`. This is exactly what we want, however, do not start Taiga yet – there is still work to be done in `taiga-front`.
+
+#### `custom-back/config.append.py`
+
+<details>
+<summary>Click here to expand</summary>
+
+```python
+INSTALLED_APPS += ["taiga_contrib_ldap_auth_ext"]
+
+LDAP_SERVER = "ldaps://ldap.example.com"
+LDAP_PORT = 636
+
+LDAP_BIND_DN = "CN=SVC Account,OU=Service Accounts,OU=Servers,DC=example,DC=com"
+LDAP_BIND_PASSWORD = "verysecurepassword"
+
+LDAP_SEARCH_BASE = 'OU=DevTeam,DC=example,DC=net'
+
+LDAP_USERNAME_ATTRIBUTE = "uid"
+LDAP_EMAIL_ATTRIBUTE = "mail"
+LDAP_FULL_NAME_ATTRIBUTE = "givenName"
+
+LDAP_SAVE_LOGIN_PASSWORD = False
+
+LDAP_MAP_USERNAME_TO_UID = None
+```
+
+Change the following fields matching your setup:
+
+**`LDAP_SERVER` and `LDAP_PORT`:** You will definitely have to change the server URL. If possible, try to keep the `ldaps://` to use a secure connection. The port can likely stay as is, unless...
+
+* ... you run the LDAP server on a different (non-standard) port.
+* ... you want to use unencrypted, insecure LDAP: In this case, change `ldaps://` to `ldap://` and the port to 389.
+* ... you want to use STARTTLS. In this case, you have to make the same changes as for unencrypted, insecure LDAP and set `LDAP_START_TLS = True`, making the section look like this:
+    ```python
+    LDAP_SERVER = "ldap://ldap.example.com"
+    LDAP_PORT = 389
+    LDAP_START_TLS = True
+    ```
+    What happens is that an unencrypted connection is established first, but then upgraded to a secure connection. To the best of my knowledge, this should also be safe – however, I like the `ldaps://` variant more.
```

Comment:
Not exactly accurate: LDAPS will use a "LDAP SSL" connection, while enabling `LDAP_START_TLS ` will use a "LDAP over TLS" connection.
Both are secured, and in fact I believe TLS is recommended, but not all LDAP server support easily Start TLS so we provide both solutions.